go/libkb/services.go

```diff
@@ -15,6 +15,10 @@ import (
 //=============================================================================
 
 func MakeProofChecker(c ExternalServicesCollector, l RemoteProofChainLink) (ProofChecker, ProofError) {
+	if c == nil {
+		return nil, NewProofError(keybase1.ProofStatus_UNKNOWN_TYPE,
```

Comment:
Yeah I added a SetServices() call in `bind/keybase.go`.

I added this to prevent a panic if services weren't configured.